src/components/position.js

```diff
@@ -1,8 +1,8 @@
-var coordinatesMixin = require('../utils/coordinates').componentMixin;
 var registerComponent = require('../core/component').registerComponent;
-var utils = require('../utils/');
 
-module.exports.Component = registerComponent('position', utils.extend({
+module.exports.Component = registerComponent('position', {
+  type: 'coordinate',
```

Comment:
I agree on that, ```vector3``` probably is more clear, but if we're going to use the same schema for shaders too, probably ```vec3``` could be better (the same way for ``ìnt``` or ```integer``` and so on)